content/open-source/apko/reference.md

```diff
@@ -83,15 +126,77 @@ This section defines users and groups that should be added to this image.
+| `run-as`      | The default user to run the entrypoint command.                                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| `uid`         | UID to associate with the file                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `gid`         | GID to associate with the file                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| `permissions` | File permissions to set. Permissions should be specified in octal e.g. 0o755 (see `man chmod` for details).                                                                                                                                                                                                                                                                                                                                                                                 |
+| `source`      | Used in `hardlink` and `symlink`, this represents the path to link to.                                                                                                                                                                                                                                                                                                                                                                                                                      |
+
+### Example
+
+```yaml
+paths:
+  - path: /app
+    type: directory
+    permissions: 0o777
+    uid: 65532
+    gid: 65532
+```
+
+## Includes
+
+Defines a path to a configuration file which should be used as the base config. By default, there is no
+base config used.
+
+The path can be either a local file, or a file in a remote git repository, in the same style as Go package names and GitHub Actions.
```

Comment:
It seems like there's a mix of long paragraphs broken up into multiple lines and others being held to one line. Not sure if it matters :shrug:

Reply:
I think I fixed all of them